### ISSUE TITLE
How to fix this problem?

### ISSUE THREAD

**Issue by Jackson8787** [NUSTM/VLP-MABSA#15]
I got the error message below when running   sh 15_pretrain_full.sh.

> OSError: Model name './E2E-MABSA' was not found in tokenizers model name list (facebook/bart-base, facebook/bart-large, facebook/bart-large-mnli, facebook/bart-large-cnn, facebook/bart-large-xsum, yjernite/bart_eli5). We assumed './E2E-MABSA' was a path, a model identifier, or url to a directory containing vocabulary files named ['vocab.json', 'merges.txt'] but couldn't find such vocabulary files at this path or url.

How can I fix it?
![スクリーンショット 2023-01-08 000312](https://user-images.githubusercontent.com/85683870/211159733-e765755a-39a5-4ff6-9150-8c0562f9613f.png)

**Comment by lyhuohuo** [comment]
This error seems the model can't find the initial bart model path, you can download the bart model files from hugging face and modify the path.

**Comment by lyhuohuo** [comment]
![image](https://user-images.githubusercontent.com/32841247/211474768-e0f46cfa-69e8-4cab-96d0-21c22dcab7b9.png)
you need to change the parameter "pretrained_model_name" as the path of your download bart model.
